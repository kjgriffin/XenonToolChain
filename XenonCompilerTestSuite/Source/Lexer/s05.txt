this is a test // of comments
that// are inline
// and start at a line
and keep going