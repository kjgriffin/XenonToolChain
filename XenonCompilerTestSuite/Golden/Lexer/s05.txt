this
is
a
test
that
and
keep
going